### PR TITLE
Pr/shorten error messages

### PR DESCRIPTION
**Pull request by joeynimu**
This PR is for shortening error messages in the console.
It simply wraps the errors in a <detail /> tag and gets the first line of the error and inserts it in a <summary /> tag. And if there are more lines in the error apart from the first line, it spits then just after the <summary /> tag inside the <detail /> tag. This enables the user to expand on the error and avoids the console being clattered.

### CODE REVIEW

**Comment by joeynimu** [comment]
@CompuIves please have a look if this is good to go, thanks.

**Comment by CompuIves** [comment]
This is super! This looks good, I found one minor issue:

screen shot 2018-03-03 at 14 47 13

It seems the line breaks are gone in the detail message. When that goes in it's good to go!

File: packages/app/src/app/components/Preview/DevTools/Console/Message/index.js [hunk 1]
```diff
@@ -50,6 +59,24 @@ function getMessage(message) {
...
+      const firstLine = splitMessage(msgLine);
+      const msgArray = msgLine.split('\n');
+      if (msgArray.length > 1) {
+        otherErrorLines = msgArray.slice(1);
+      }
+      return (
+        <InnerItem>
+          <details>
+            <summary>{firstLine}</summary>
+            {otherErrorLines}
```
Comment (CompuIves) [inline packages/app/src/app/components/Preview/DevTools/Console/Message/index.js]: I think this should become `otherErrorLines.join('\n\r')` to make it work again 😄

File: packages/app/src/app/components/Preview/DevTools/Console/Message/index.js [hunk 2]
```diff
@@ -50,6 +59,24 @@ function getMessage(message) {
   }
 
   if (message.arguments.every(argument => typeof argument === 'string')) {
+    if (message.logType === 'error') {
```
Comment (CompuIves) [inline packages/app/src/app/components/Preview/DevTools/Console/Message/index.js]: Would also be good to put a comment here what the purpose of this code is. For new readers.

**Comment by CompuIves** [comment]
Works perfectly! Thank you very much 😄

**Comment by joeynimu** [comment]
Awesome!

**Pull request merged**